src/fty_common_rest_utils_web.cc

```diff
@@ -207,9 +209,22 @@ std::string escape (const char *string) {
         else if (c == '\\') {
             after.append ("\\\\");
         }
+	// escape UTF-8 chars which have more than 1 byte
+	else if (width > 1) {
+	    // allocate memory for "\u" + 4 hex digits + terminator
+	    char *codepoint = (char *) calloc (7, sizeof (char));
```

Comment:
Just a few thoughts:
* could it be faster to allocate a `2^N` buffer so 8 bytes here?
* are we so uncertain about `UTF8::utf8_to_codepoint()` overflowing this buffer, and by just one byte at that? Do we want to `assert(codepoint[6] == '\0')` (to notice bugs) instead of just forcing the zero ending again (note that `calloc` should have initialized the buffer to zeroes already, although pedantically a `0` is not `\0` :) )?

Reply:
After discussion, we came to a conclusion that performance improvement is insignificant.

Reply:
...and that it might be a bit faster on 64-bit platforms, so not likely a bonus for our current HW ARM appliances but can be a bonus for x86_64 builds and if we use this codepath extensively (international users and corresponding asset names, etc).